hw1/02-palindrome.js

```diff
@@ -1,3 +1,46 @@
-const elem = document.querySelector('input');
+const inputElem = document.querySelector("input");
+const NOT_A_NUMBER = "Not a number.";
+const POSITIVES_ONLY = "Input cannot be less than 0.";
```

Comment:
The Airbnb documentation mentions using single quotes for strings **Rule 6.1**. However, as mentioned in class, prettier automatically saves them as double quotes and the professor isn't exactly looking for this precise integration.

Reply:
That's why I brought it up. And the rule doesn't even state why but the ES standard just says "be consistent with the usage" for how it's being used. The single vs double is probably just the standard they (Airbnb) wanted to use in their code base. Thanks for taking a look.